packages/clarity-js/src/data/metric.ts

```diff
@@ -1,8 +1,10 @@
-import { Event, Metric, MetricData } from "@clarity-types/data";
+import { Code, Constant, Event, Metric, MetricData, Severity } from "@clarity-types/data";
+import * as log from "@src/diagnostic/log";
 import encode from "./encode";
 
 export let data: MetricData = null;
 export let updates: MetricData = null;
+const regex = /[^0-9\.]/g;
```

Comment:
Minor: Doesn't' work with non-US numbers. Suggest you allow comma (,) as well.

Reply:
Fixed thanks.